stroyprombeton/models.py

```diff
@@ -44,6 +44,17 @@ def get_absolute_url(self):
 class Option(models.Model):
     """This doc page describes what is option: https://goo.gl/S4U9PG."""
 
+    @property
+    def name(self):
+        return self.product.name
+
+    def get_absolute_url(self):
+        return self.product.url
+
+    @property
+    def url(self):
+        return self.product.url
+
```

Comment:
Let's provide `page` property here too. It is required for cart's position images